spec/models/case_log_spec.rb

```diff
@@ -332,17 +350,17 @@
           end
 
           it "correctly derives and saves weekly service charge" do
-            case_log.update!(scharge: 100, period: 2)
+            case_log.update!(scharge: 70, period: 2)
```

Comment:
Why did you have to change the value of the floats in the weekly charges tests?

Reply:
Because the original ones were too high for the organisation type but that wasn't being set so the validation wasn't triggering. 

Calculation being tested should be working the same though.